developers/wallets/index.md

```diff
@@ -66,21 +50,66 @@ In order to register a credential handler, your web app must serve a "manifest.j
   target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/manifest.json </a>
 </p>
 
-### 3. Allow your users to register their wallet's credential handler with the browser polyfill
+The `/wallet-worker.html` page will setup listeners for CHAPI-sent events
```

Comment:
```suggestion
The `/wallet-worker.html` page (note: you can use another name if you'd like) will setup listeners for CHAPI-sent events
```

Reply:
I did have something like that at one point...I'll put it back.